services/AUTOMATIC1111/clone.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -Eeuo pipefail
```

Comment:
can we also add `x` to flags? just like the shell

Reply:
Yep, will change